website/cue/reference/releases/0.45.0.cue

```diff
@@ -0,0 +1,332 @@
+		{
+			type: "feat"
+			description: """
+				Query parameters can now contain either `single value` or array of `multiple values`.
+				"""
+			contributors: ["sainad2222"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add a new type of `enrichment_table` - `memory`, which can also act as a sink, taking in all the
+				data and storing it per key, enabling it to be read from as all other enrichment tables.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "feat"
+			description: """
+				A new sink for [Keep](\(urls.keep)) was added.
+				"""
+			contributors: ["sainad2222"]
+		},
+		{
+			type: "feat"
+			description: """
+				The `host_metrics` source has a new collector, `tcp`. The `tcp`
+				collector exposes three metrics related to the TCP stack of the
+				system:
+
+				* `tcp_connections_total`: The total number of TCP connections. It
+				  includes the `state` of the connection as a tag.
+				* `tcp_tx_queued_bytes_total`: The sum of the number of bytes in the
+				   send queue across all connections.
+				* `tcp_rx_queued_bytes_total`: The sum of the number of bytes in the
+				  receive queue across all connections.
+
+				This collector is enabled only on Linux systems.
+				"""
+			contributors: ["aryan9600"]
+		},
+		{
+			type: "fix"
+			description: """
+				The `chronicle_unstructured` sink now sets the `content-encoding` header when compression is enabled.
+				"""
+			contributors: ["chocpanda"]
+		},
+		{
+			type: "feat"
+			description: """
+				The `tag_cardinality_limit` transform now supports customizing limits for specific metrics, matched by metric name and optionally its namespace.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add `websocket_server` sink that acts as a websocket server and broadcasts events to all clients.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "feat"
+			description: """
+				Sources running HTTP servers (`http_server` source, `prometheus` source, `datadog_agent`, etc.) now support a new `custom` authorization strategy . If a strategy is not explicitly defined, it defaults to `basic`, which is the current behavior.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "fix"
+			description: """
+				The systemd service now validates the config with parameter `--no-environment` on service reload.
+				"""
+			contributors: ["rsrdesarrollo"]
+		},
+		{
+			type: "feat"
+			description: """
+				The `dnstap` source now uses [v20250201](https://github.com/dnstap/dnstap.pb/releases/tag/v20250201) dnstap protobuf schema.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "fix"
+			description: """
+				Allow users to specify `session_name` when using aws auth.
+				"""
+			contributors: ["akutta"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Add support for more chronicle regional endpoints as listed - https://cloud.google.com/chronicle/docs/reference/ingestion-api#regional_endpoints
```

Comment:
```suggestion
				Add support for more Google Chronicle regional endpoints: <list of endpoints>.
```

Could we just list out the added endpoints so people don't have to guess what was added?